packages/iframe-coordinator/src/HostRouter.ts

```diff
@@ -13,9 +13,14 @@ export class HostRouter {
   private _clients: ClientInfo[];
 
   constructor(clients: RoutingMap) {
-    this._clients = Object.keys(clients).map((id) => {
-      return parseRegistration(id, clients[id]);
-    });
+    this._clients = Object.entries(clients)
```

Comment:
Noticed that we were doing essentially Object.entries and doing brackets to get array data, which I've seen flagged before on code analyzers.